src/views.js

```diff
@@ -0,0 +1,126 @@
+import { addDomElem } from './helper';
+import { myProjects } from './project-model';
+import { box } from './forms';
+
+const taskListDiv = addDomElem('div', 'class', 'mt-3');
+const projectsListDiv = addDomElem('div', 'class', 'mt-3'); // Inside this the boxes with every project
+
+// Render each project *third menu button*
+const renderProjectItems = () => {
+  const template = document.getElementById('template-project').content;
+  const fragment = document.createDocumentFragment();
+  const projectDiv = addDomElem('div', 'class', 'alert alert-danger d-flex align-items-center');
+  projectDiv.classList.add('justify-content-between');
+  projectDiv.setAttribute('role', 'button');
+  const projectName = addDomElem('p', 'class', 'm-0');
+  // const iconH3 = addDomElem('h3', 'class', 'm-0');
+  // const trashIcon = addDomElem('i', 'class', 'fa fa-times-circle text-danger');
+  // trashIcon.setAttribute('role', 'button');
```

Comment:
- [ ] [REQUIRED] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
**PS:** Please remove all comments in this file.